docs/tutorials/dlinktree-builder.md

```diff
@@ -260,23 +260,19 @@ async function assembleCode() {
 
     let pageTitle = document.getElementById("title").textContent;
 
-    let basicCSS = `
-    @import url("agregore://theme/vars.css");
-                body, * {
-                    font-size: 1.2rem;
-                    margin: 0;
-                    padding: 0;
-                    font-family: var(--ag-theme-font-family);
-                    background: var(--ag-theme-background);
-                    color: var(--ag-theme-text);
-                }
-    `;
+    // Fetch styles.css content to import as our basic CSS
```

Comment:
Sorry I meant to import the agregore theme's style at `agregore://theme/style.css` instead of just the vars. Else I like the inline css

Reply:
Of course! I notice now that I've been only importing the vars. I will update these.